src/extensions/filter-control/utils.js

```diff
@@ -45,8 +59,14 @@ export function existOptionInSelectControl (selectControl, value) {
 }
 
 export function addOptionToSelectControl (selectControl, _value, text, selected) {
-  const value = (_value === undefined || _value === null) ? '' : _value.toString().trim()
-  const $selectControl = $(selectControl.get(selectControl.length - 1))
+  let value = (_value === undefined || _value === null) ? '' : _value.toString().trim()
+
+  value = value.replace(/(<([^>]+)>)/ig, '').replace(/&[#A-Za-z0-9]+;/gi, '').trim()
```

Comment:
is'nt it possible to merge that to regex ?
Why the first regex hast two capture groups for replacing ?
In the second regex we have  `a-z`, `A-Z` and the flag `i` which makes the case insensitive. So we can remove one of the `a-z`/`A-Z` or the i flag.